example/home/test/test_blog.py

```diff
@@ -464,6 +469,60 @@ def test_blog_body_pagechooserblock(self):
         # Check that we test all blocks that were returned.
         self.assertEquals(len(query_blocks), count)
 
+    def test_blog_body_snippetchooserblock_advert(self):
+        url = "https://http.cat"
+        text = "cats"
+        blog_page = BlogPageFactory(
+            parent=self.home,
+            body=[
+                ("advert", AdvertFactory(url=url, text=text)),
+            ],
+        )
+        block_type = "SnippetChooserBlock"
+        block_query = """
+        snippet {
+            ... on Advert {
+                url
+                text
+            }
+        }
+        """
+        query_blocks = self.get_blocks_from_body(
+            block_type, block_query=block_query, page_id=blog_page.id
+        )
+        self.assertEqual(len(query_blocks), 1)
+        block = query_blocks[0]
+        self.assertEqual(block["blockType"], block_type)
```

Comment:
Question: Would we expect `get_blocks_from_body()` to ever return anything but `SnippetChooserBlock` value types when that is supplied as the first argument? I can fully understand the value of the following two assertions, but this one, I'm not so sure about.

Reply:
Yep, I think you're correct, it's unnecessary.